common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -28,7 +28,9 @@ class ProjectCard extends React.PureComponent<Props> {
     return (
       <a style={styles}
         className="ProjectCard-root"
-        href={url.section(Section.AboutProject, {id: this.props.project.id})}>
+        href={url.section(Section.AboutProject, {id: this.props.project.id})}
```

Comment:
I'd like this to be revisted by design after the double history entry bug currently open in #63 is solved. Unless we have specific testing otherwise, [this sums up my position](https://medium.com/the-metric/links-should-open-in-the-same-window-447da3ae59ba)

Reply:
It was a team decision, so I'd bring this up with design.